File: Domain/Review.cs
namespace Domain
{
	public class Review
	{
		public Guid Id { get; set; }
		public string Body { get; set; }
		public int Rating { get; set; }
		public AppUser Author { get; set; }
		public HigherEducationFacility HigherEducationFacility { get; set; }
		public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
	}
}

File: client-app/src/app/models/review.ts
export interface Review {
	id: string;
	createdAt: Date;
	rating: number;
	body: string;
	username: string;
	fullName: string;
	image: string;
}

export class ReviewFormValues {
	id: string = "";
	rating: number = 0;
	body: string = "";
}